### PR TITLE
DATAGO-4844: Account deletion for Standard accounts

### PR DESCRIPTION
**Pull request by shwetaarshad**

### CODE REVIEW

File: docs/group_quick_starts/gc_account_deletion.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Account Deletion
+================================
+
+You can now delete your account without asking help from Solace Cloud support. However, this feature is limited to standard accounts only.
+
+In the below image, under Account tab, the Advance Setting link is been added to provide option for deleting account.
```
Comment (ianmwong) [inline docs/group_quick_starts/gc_account_deletion.rst]: Maybe change this to:

> Under the Account tab, click on the Advanced Settings link to view the delete account button as shown below.

Comment (shwetaarshad) [reply docs/group_quick_starts/gc_account_deletion.rst]: Changed.

File: docs/group_quick_starts/gc_account_deletion.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. figure:: ../img/delete_account_advanced_settings.png
+
+
+.. figure:: ../img/delete_account_popup.png
+
+After clicking the Delete Account button you would get a confirmation pop up which will inform you about all the impacts post deletion of your account.
```
Comment (ianmwong) [inline docs/group_quick_starts/gc_account_deletion.rst]: Suggested text:

> After clicking on the Delete Account button, a confirmation dialog will be displayed to provide you with an overview of the components of your Solace Cloud PubSub+ account that will be deleted. You will then be prompted to enter your password to confirm the deletion.

File: docs/group_quick_starts/gc_account_deletion.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.. figure:: ../img/delete_account_confirmation_popup.png
+
+You would require to enter your password to ensure that it is an intentional deletion and voila, the account is deleted.
```
Comment (ianmwong) [inline docs/group_quick_starts/gc_account_deletion.rst]: I don't think you need this paragraph or the image.

File: docs/group_quick_starts/gc_account_deletion.rst (hunk 2 above)
Comment (shwetaarshad) [reply docs/group_quick_starts/gc_account_deletion.rst]: Incorporated.

File: docs/group_quick_starts/gc_account_deletion.rst (hunk 3 above)
Comment (shwetaarshad) [reply docs/group_quick_starts/gc_account_deletion.rst]: As discussed took out the other screenshot which shows only Delete button with password.

File: docs/group_quick_starts/gc_account_deletion.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+.. figure:: ../img/delete_account_success_msg.png
+
+Once the account is deleted, all of your messaging services will be terminated, your outstanding bill will be emailed to you and your credit card would be charged as per the usage.
```
Comment (ianmwong) [inline docs/group_quick_starts/gc_account_deletion.rst]: > Once the account is deleted, all of your messaging services will be terminated, your outstanding bill will be emailed to you, your credit card will be charged, and you will be logged out of the system.

>If you require any assistance after your account has been deleted, please contact our Support team.

@mikesmith40 , comments?

Comment (shwetaarshad) [reply docs/group_quick_starts/gc_account_deletion.rst]: @ianmwong how about this one:

Once the account is deleted, all of your messaging services will be terminated and you will be logged out of the system. Your outstanding bill will be emailed to you and your credit card would be charged. 

If you require further assistance after your account has been deleted, please contact our Support team.

**Pull request merged**